draft-ietf-core-oscore-groupcomm.md

```diff
@@ -720,9 +720,11 @@ The occurrence of such an event and how long it would take to occur depend on th
 
    -  The Group Manager MUST evict the elder members from the group. That is, the Group Manager MUST terminate their membership and, in the following steps, it MUST rekey the group in such a way that the new keying material is not provided to those evicted elder members.
 
-      This ensures that: i) an Observe notification {{RFC7641}} can never successfully match against the Observe requests of two different observations; and ii) a non-notification response can never successfully match against the group requests of two different Non-Notification Group Exchanges. In fact, the excluded elder members would eventually re-join the group, thus terminating any of their ongoing (long-lasting) observations (see {{sec-long-term-observations}}) and Non-Notification Group Exchanges (see {{sec-replay-protection-non-notifications}}).
+      This ensures that any response from the same server to the request of a long exchange can never successfully match against the request of two different long exchanges.
+
+      In fact, the excluded elder members would eventually re-join the group, thus terminating any of their ongoing long exchanges (see {{sec-long-exchanges}}).
```

Comment:
Why "would" the elder member eventually re-join? Is it necessarily so, or should it be "could"?

Reply:
"could" is fine